content/blog/2017/2017-01-12-scm-api-2.0-release.adoc

```diff
@@ -0,0 +1,230 @@
+---
+:layout: post
+:title: SCM API turns 2.0 and what that means for you
+:tags:
+- development
+- plugins
+:author: stephenc
+---
+
+We've made some great improvements to a number of popular plugins including:
+
+* GitHub Branch Source
+* BitBucket branch source
+* Git
+* Mercurial
+* Branch API
+* SCM API
+* Pipeline Multibranch
+* GitHub Organization Folders
+
+There are some gotcha's that Jenkins administrators will need to be aware of.
+
+We want to give you the whole story, but the <<tldr,TL;DR>> is this
+
+TIP: When updating your plugins, please be sure to upgrade the GitHub Branch Source and/or Bitbucket branch source plugins.
```

Comment:
For each plugin listed can we include a hyperlink to the description?

GitHub Branch Source - https://plugins.jenkins.io/github-branch-source
BitBucket branch source - https://plugins.jenkins.io/cloudbees-bitbucket-branch-source
Git - https://plugins.jenkins.io/git
Mercurial - https://plugins.jenkins.io/mercurial
Branch API - https://plugins.jenkins.io/branch-api
SCM API - https://plugins.jenkins.io/scm-api
Pipeline Multibranch - https://plugins.jenkins.io/workflow-multibranch
GitHub Organization Folders - https://plugins.jenkins.io/github-organization-folder